src/bin/deku_node.ml

```diff
@@ -150,5 +156,5 @@ let node =
     let open Arg in
     required & pos 0 (some string) None & info [] ~doc ~docv in
   let open Term in
-  const node $ folder_node
+  const node $ folder_node $ Prometheus_dream.opts
```

Comment:
Add the `--listen-prometheus=<port>` option, if it's not given we won't start the prometheus server